vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinConfigurationProperties.java

```diff
@@ -112,39 +112,83 @@ public void setPnpmEnabled(boolean pnpmEnabled) {
     }
 
     /**
-     * Get a list of packages that are blacklisted for class scanning.
+     * Get a list of packages that are blocked for class scanning.
      *
-     * @return package blacklist
+     * @return blocked packages
+     * @deprecated use getBlockedPackages()
      */
+    @Deprecated
```

Comment:
It would be good to add the `forRemoval=true` attribute

Reply:
Done.